### PR TITLE
chore(lib/common): Use Uint128 from scale instead of lib/common

### PR DESCRIPTION
**Pull request by kishansagathiya**
## Changes

- Removed uint128.go
- Replaced common.Uint128 with scale.Uint128

## Tests

`make gossamer` and `go test ./lib/babe` should be enough

## Issues

- #1939

## Primary Reviewer

- @timwu20

### CODE REVIEW

**Review by noot — APPROVED** [review]
looks good!

**Review by timwu20 — APPROVED** [review]
LGTM!

File: lib/babe/crypto.go [hunk 1]
```diff
@@ -58,25 +62,28 @@ func claimPrimarySlot(randomness Randomness,
...
-) bool {
+) (bool, error) {
 	t := makeTranscript(randomness, slot, epoch)
 	inout := sr25519.AttachInput(output, pub, t)
 	res := sr25519.MakeBytes(inout, 16, babeVRFPrefix)
 
-	inoutUint := common.Uint128FromLEBytes(res)
+	inoutUint, err := scale.NewUint128(res)
+	if err != nil {
+		return false, fmt.Errorf("failed to convert bytes to Uint128, %w", err)
```
Comment (qdm12) [inline lib/babe/crypto.go:74]: nit I know it's a super detail, but should we agree on a common way to wrap errors? I tend to use `context: %w` which seems like the common standard out there, but I'm happy to use any wrapping, as long as we make it the same in this codebase.

**Review by qdm12 — APPROVED** [review]
:+1: Nice!

File: lib/babe/crypto.go (hunk 1 above)
Comment (kishansagathiya) [reply lib/babe/crypto.go:74]: will change it :+1:

Comment (kishansagathiya) [reply lib/babe/crypto.go]: done

**Pull request merged**